presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -417,6 +427,29 @@ private static SliceWriteFunction typedVarcharWriteFunction(String jdbcTypeName)
         };
     }
 
+    private static SliceWriteFunction uuidWriteFunction()
+    {
+        return (statement, index, value) -> {
+            long mostSig = value.getLong(0);
```

Comment:
Nit: let's inline `mostSig` and `leastSig` (like we do in `UuidType` and `UuidOperators`)